ERCS/erc-7715.md

```diff
@@ -0,0 +1,466 @@
+---
+eip: 7715
+title: Request Permissions from Wallets
+description: Adds JSON-RPC method for requesting permissions from a wallet
```

Comment:
Editorial: Can you clarify "who" requesting permissions from wallet? It's both important from a description to provide the intended requester as design target but also to imply how does wallet identify who IS being approved, which is a technical question. 

Techincal: I will ask the question as a peer reviewer in the discussion thread